services/workers/workers.conf.dev.yml

```diff
@@ -0,0 +1,26 @@
+stackName: "${env:PROJECT_NAME}-workers-${self:provider.stage}"
```

Comment:
I didn't know there was an option to modify stack name! this is good news.
Can you make this one follow the same pattern as other stacks do?
`${env:PROJECT_NAME}-${env:ENV_STAGE}-workers` ?

Be sure to update CodeBuild roles in `ciServerless.ts`!